libraries/tickv/src/lib.rs

```diff
@@ -205,10 +205,10 @@
 
 #![no_std]
 #![deny(unsafe_code)]
-#![deny(missing_docs)]
+//#![deny(missing_docs)]
```

Comment:
We shouldn't remove this

Reply:
The best bet is just to add documentation to the functions. I'll try and do it for you

Reply:
This will do it: https://github.com/tock/tock/pull/3090

Reply:
Thank you for your help!

Reply:
I copied the code you made pull request, and merged that code into my local work directory. It works well! Thank you.